File: questão3.py
nomes = []
idades = []

for i in range(3):
    nome = input(f"Digite o nome da {i+1}ª pessoa: ")
    nomes.append(nome)

    idade = int(input(f"Digite a idade da {i+1}ª pessoa: "))
    idades.append(idade)

maiorIdade = idades[0]
maisVelho = nomes[0]

for i in range(3):
    if(idades[i] > maiorIdade):
        maiorIdade = idades[i]
        maisVelho = nomes[i]

print(f'O mais velho  é {maisVelho} com {maiorIdade} anos')

File: questão3.c
#include <stdio.h>
#include <stdlib.h>

#define MAX_LENGTH 100

int main(){
    int quantidadePessoas;
    printf("Digite a quantidade de pessoas que vai ser cadrastadas: ");
    scanf("%d", &quantidadePessoas);

    char nomes[quantidadePessoas][MAX_LENGTH];
    int idades[quantidadePessoas];

    for (int i = 0; i < quantidadePessoas; i++){
        for (int j = 0; j < 1; j++){
            printf("Digite o nome da pessoa %d: ", i+1);
            scanf("%s", &nomes[i][j]);
            printf("Digite a idade de %d: ", i+1);
            scanf("%d", &idades[i]);
    }}

    int maisVelho = idades[0];

    int cont = 0;
    for (int i = 0; i < quantidadePessoas; i++){
        if (maisVelho < idades[i]){
                maisVelho = idades[i];
                cont+=1;
        }
    }


    printf("O mais velho eh o/a %s", nomes[cont]);
    return 0;
}

